src/api.js

```diff
@@ -0,0 +1,49 @@
+var _ = require('underscore');
+var createDashboard = require('./create-dashboard');
+var cdb = require('cartodb.js');
+
+function Dashboard (dashboard) {
+  this.dashboard = dashboard;
+
+  // bind widget related methods
+  var ws = dashboard.widgets;
+  this.createCategory = ws.createCategoryModel.bind(ws);
```

Comment:
I like this API object :+1: 

Since these methods are going to be public (as in API public), I'd be nice to declare them outside of the constructor and document them (something basic):
```
/**
 * Create a category widget.
 * @param {Object} widgetAttrs - attributes for the new widget
 * @param {string} widgetAttrs.id - id (required)
 * @param {string} widgetAttrs.title - title (required)
 * @param {number} widgetAttrs.order - index of the widget (optional)
 * @param ...
 * @return {CategoryWidget} The new widget
 */
Dashboard.prototype.createCategoryWidget = function (widgetAttrs) {
  this._dashboard.widgets.createCategoryModel.call(this, widgetAttrs);
}
```

Perhaps this would help us generate some nice docs?